CONTRIBUTING GUIDELINES.md

```diff
@@ -0,0 +1,106 @@
+<img src="https://phoenixnap.com/kb/wp-content/uploads/2019/12/location-git-windows-download.png" width=600>
+**4.** Allow the app to make changes to your device by clicking Yes on the User Account Control dialog that opens.<br>
+**5.** Review the GNU General Public License,and the installer will ask you for an installation location. Leave the default, unless you have reason to change it.<br>
+**7.** Then installer will install  Git with default settings<br>
+**8.** Then create a GitHub account [here](https://github.com/)<br>
+
+<h2 align="center"><b>Pull Request Process</b></h2> 
+**1.**  Fork [this](https://github.com/smaranjitghose/awesome-portfolio-websites.git) repository.
+**2.**  Clone the forked repository.
+
+```
+git clone --depth 1 https://github.com/<your_user_name>/awesome-portfolio-websites.git
+```
+   <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcT5N0HJ9db7jSvcL4dsDscZQBzqQqqKVs0BnO1OVz26glLWKJRY&usqp=CAU" width="300">
+**3.** Navigate to the project directory.
+
+```
+cd awesome-portfolio-websites
+```
+**4.** Add a reference to the original repository.
+
+```
+git remote add upstream  https://github.com/smaranjitghose/awesome-portfolio-websites.git (url of the repository)
+```
+
+**5.** See latest  changes to the repo using
+
+```
+git remote -v
+```
+
+**6.** Create a new branch.
+
+```
+git checkout -b <your_branch_name>
+```
+
+**7.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
+```
+git pull upstream master
+```
+
+**8.** Make changes in the source code after getting yourself assigned to the issue by the mentors/developers of the project.
+
+<p align="center"><img width=35% src="https://media2.giphy.com/media/L1R1tvI9svkIWwpVYr/giphy.gif?cid=ecf05e47pzi2rpig0vc8pjusra8hiai1b91zgiywvbubu9vu&rid=giphy.gif"></p>
+
+**9.** Add and then commit your changes.
+
+```
+git add . or git add --all
+```
+```
+git commit -m "<your_commit_message>"
+```
+
+
+**10.** Push your local branch to the remote repository.
+
+```
+git push -u origin <your_branch_name>
+```
+
+**11.** Create a Pull Request!
+
+**12.** Finally, go to your repository in browser and click on `compare and pull requests`.
+
+**13.** Then add an appropriate title and description to your pull request that explains your changes and efforts done.
+
+   <img src="https://user-images.githubusercontent.com/41269164/70219707-47194780-176b-11ea-96c2-d0c401ddb1e0.png" width=600>
+
+**14.** Click on `Create Pull Request`.
+
+
+   <img src="https://user-images.githubusercontent.com/41269164/70219836-8d6ea680-176b-11ea-81d5-549093bf0954.png" width=600>
+
+<h2 align="center"><b>Congrats for your Contribution!!! </b></h2>
+<p align="center"><img scr="https://media2.giphy.com/media/rY93u9tQbybks/giphy.gif?cid=ecf05e4717631a3f454f724747f59e0cba22aa1d1f534d47&rid=giphy.gif" width=500"></p>
+
```

Comment:
15. Voila! You have made a PR to the awesome-developer-portfolio project. Sit back patiently and relax while the project maintainers review your PR. Please understand at times the time can vary from a few hours to a few days